src/site/content/en/fast/serve-images-with-correct-dimensions/index.md

```diff
@@ -121,6 +122,29 @@ If you'll be resizing many images, you may find it more convenient to use a
 script or service to automate the process. You can learn more about this in the
 Responsive Images guide.
 
+## Avoid layout shifts by specifying dimensions
+
+While this guide discusses image dimensions in the context of performance, it's important to note that reserving space for images in the layout is a crucial component of minimizing a page's [Cumulative Layout Shift](/cls/) metric. When serving images in HTML, be sure to use proper `width` and `height` attributes so that the browser knows how much space to allocate in the layout for the image:
+
+```html
+<img src="flower.jpg" width="640" height="480" alt="A picture of a siberian iris.">
+```
+
+Without these attributes, the browser has no idea how much space the image will take up until it has loaded. This will cause layout shifts in the document, which can be frustrating to users when they "miss" their intended hit target and end up clicking on something else they didn't intend to during page load.re-run
+
+If you're using images in CSS, be sure to use the [`aspect-ratio` property](https://developer.mozilla.org/docs/Web/CSS/aspect-ratio) on the image's container. This has the same effect on an element's size that `width` and `height` attributes do in the sense that the container will maintain a consistent aspect ratio.
+
+```css
+.masthead {
+  background: url('/flower.jpg');
+  aspect-ratio: 16 / 9;
+}
```

Comment:
I find this section weird in this context and think we should remove it.

A CSS background-image will not resize the element after the image downloads to match the image's dimensions. Therefore the `aspect-ratio: 16 / 9;` is needed to size the element anyway. There will be no CLS without it (though the image also won't be shown!).

You can use `aspect-ratio` on an `img` element (Jake talked about it recently in this post: https://jakearchibald.com/2022/img-aspect-ratio/), but that's not as a background image and doesn't have quite "the same effect on an element's size that `width` and `height` attributes do".

So maybe rephrase it as:

```suggestion
An alternative to providing the width and height explicitly, is to use the  CSS [`aspect-ratio` property](https://developer.mozilla.org/docs/Web/CSS/aspect-ratio) on the image. This has a similar effect on an element's size that `width` and `height` attributes do in the sense that the container will maintain a consistent aspect ratio. However, the difference is that this may result in a different aspect-ratio being used than the image is provided in, so you will likely want to use an `object-fit` setting to ensure the image is not distorted in this explicit 16/9 view:

```css
img {
  aspect-ratio: 16 / 9;
  width: 100%;
  object-fit: cover;
}
```